packages/mirage-block-ramdisk/mirage-block-ramdisk.0.6/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: "David Scott"
+license: "ISC"
+tags: "org:mirage"
+homepage: "https://github.com/mirage/mirage-block-ramdisk"
+doc: "https://mirage.github.io/mirage-block-ramdisk/"
+bug-reports: "https://github.com/mirage/mirage-block-ramdisk/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune"
+  "alcotest" {with-test}
+  "cstruct" {>= "6.0.0"}
+  "io-page" {>= "2.4.0"}
+  "mirage-block" {>= "2.0.0"}
+  "mirage-block-combinators" {with-test}
+  "lwt"
+  "fmt" {with-test & >= "0.8.7"}
+]
+build: [
+ [ "dune" "subst" ] {dev}
+ [ "dune" "build" "-p" name "-j" jobs ]
+ [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
```

Comment:
Given the discussion, should this include the availability constraint?

```suggestion
]
available: os-family != "windows"
```

Reply:
I'm not sure what the aim is for opam-repository, but take into account that what fails to install is io-page, a dependency of mirage-ramdisk.... so my preference would be to add such availability constraints to io-page (i.e. the package that actually fails to install).

but as said I've no clue what the opam-repository policy on such failures is.

Reply:
Ah, I missed that the limitation here was from a dependency rather your package. My bad. I totally agree that the unavailability should cascade down from the sources of the unavailability and not be added to every package.

Could be useful to codify the policy for this.